app/experimenter/experiments/tests/api/v5/test_queries.py

```diff
@@ -745,6 +746,38 @@ def test_paused_experiment_returns_date(self):
         self.assertEqual(experiment_data["isEnrollmentPaused"], True)
         self.assertEqual(experiment_data["enrollmentEndDate"], "2021-01-08")
 
+    @parameterized.expand(
+        [
+            [NimbusExperimentFactory.Lifecycles.PAUSING_REVIEW_REQUESTED, False, True],
+            [NimbusExperimentFactory.Lifecycles.PAUSING_APPROVE, False, True],
+            [NimbusExperimentFactory.Lifecycles.PAUSING_APPROVE_WAITING, False, True],
+            [NimbusExperimentFactory.Lifecycles.PAUSING_APPROVE_TIMEOUT, False, True],
+            [NimbusExperimentFactory.Lifecycles.PAUSING_APPROVE_APPROVE, True, False],
+            [NimbusExperimentFactory.Lifecycles.PAUSING_REJECT, False, False],
+            [NimbusExperimentFactory.Lifecycles.PAUSING_APPROVE_REJECT, False, False],
+        ]
+    )
+    def test_experiment_pause_pending(self, lifecycle, expected_paused, expected_pending):
```

Comment:
Figured I might as well try exhausting all the cases here to make sure the pending vs published pause states worked as expected.